dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/datadeposit/CollectionDepositManagerImpl.java

```diff
@@ -69,7 +70,10 @@ public DepositReceipt createNew(String collectionUri, Deposit deposit, AuthCrede
         AuthenticatedUser user = swordAuth.auth(authCredentials);
         DataverseRequest dvReq = new DataverseRequest(user, request);
 
-        urlManager.processUrl(collectionUri);
+        urlManagerServiceBean.processUrl(collectionUri);
+        UrlManager urlManager = urlManagerServiceBean.getUrlManager();
+
+        this.urlManager = urlManagerServiceBean.getUrlManager();
```

Comment:
2 invocations of the same method? I don't see class field 'urlManager" used anywhere, I think it can be removed.